packages/jobs/lib/processor/handler.ts

```diff
@@ -22,13 +28,78 @@ async function abort(_task: OrchestratorTask): Promise<Result<void>> {
 }
 
 async function action(task: TaskAction): Promise<Result<JsonValue>> {
```

Comment:
when executing via Temporal the `reportFailure` function is also called in case of an exception but for actions/webhooks the only relevant logic is this telemetry logs https://github.com/NangoHQ/nango/blob/4c6e7e0b5ca2d84358d62c4a11d2c385f3877528/packages/jobs/lib/activities.ts#L476 which doesn't seem to be used (in favor of those ones https://github.com/NangoHQ/nango/blob/master/packages/shared/lib/clients/orchestrator.ts#L184). So I have decided not to called `reportFailure` then. Let me know if I am mistaken

Reply:
I think it highlights once again we need to split this in: SyncRun ActionRun WebhookRun and DryRun to clarify everything